fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs

```diff
@@ -1,11 +1,12 @@
+/// Library functions for internal use by Dark processes
 module BackendOnlyStdLib.LibDarkInternal
 
+// WHATISTHIS why does this module exist in F#, but not OCaml?
```

Comment:
This also exists in OCaml, pretty much word for word

Reply:
My mistake - I've found it now.